### PR TITLE
do not run builds twice on dependabot PRs

### PR DESCRIPTION
**Pull request by lplewa**
This PR probably works - not sure how to check it. Maybe someone who has more experience with GHa can validate it. 

ref: #584

### CODE REVIEW

**Review by lukaszstolarczuk — CHANGES_REQUESTED** [review]
You could test this on your fork, but I guess it should work - the syntax looks ok.

Nonetheless, pls update also other workflows triggered on push

**Comment by lplewa** [comment]
> You could test this on your fork, but I guess it should work - the syntax looks ok.
> 
> Nonetheless, pls update also other workflows triggered on push

done i guess.

**Comment by lukaszstolarczuk** [comment]
> > You could test this on your fork, but I guess it should work - the syntax looks ok.
> > Nonetheless, pls update also other workflows triggered on push
> 
> done i guess.

what about CodeQL?

**Comment by lplewa** [comment]
> > > You could test this on your fork, but I guess it should work - the syntax looks ok.
> > > Nonetheless, pls update also other workflows triggered on push
> > 
> > 
> > done i guess.
> 
> what about CodeQL?

done too

File: .github/workflows/bandit.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Bandit static analysis (for Python code)
 name: Bandit
-
-on: [push, pull_request]
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+    pull_request:
```
Comment (lukaszstolarczuk) [inline .github/workflows/bandit.yml:7]: PR should be aligned with push, not with "branches-..." (pls apply everywhere)

Comment (lplewa) [reply .github/workflows/bandit.yml]: done

**Review by KFilipek — APPROVED** [review]

**Review by lukaszstolarczuk — APPROVED** [review]

**Pull request merged**